tdrs-frontend/src/components/SubmissionHistory/SubmissionHistory.test.js

```diff
@@ -474,4 +486,42 @@ describe('SubmissionHistory', () => {
       }
     }
   )
+
+  it('Shows Reprocessed button', () => {
```

Comment:
@ADPennington I just tested in the Raft env since it is still deployed there and it is working as expected for TANF, SSP, and Tribal. Maybe we can deploy this back to qasp when you are ready and check it out together?

Reply:
Roger. I tested active, closed, and aggregate for TANF. Active and aggregate for SPP. Only active for Tribal. Let me know when you are ready to test it out and we can hop on a call or work async via the deployment

Reply:
@elipe17 re-deploying now